CONTRIBUTING.md

```diff
@@ -180,6 +180,9 @@ authEndpoint=https://repo-dev.dev.sagebase.org/auth/v1
 fileHandleEndpoint=https://repo-dev.dev.sagebase.org/file/v1
 ```
 
+#### Running OpenTelemetry in Integration Tests
+`tests/integration/conftest.py` is where we defining which trace provider to use. Set the `SYNAPSE_OTEL_INTEGRATION_TEST_PROVIDER` environment variable to `otlp` or `console` depending on your use case.
```

Comment:
Did we want to go into a bit of detail of what to add, or is that more on a per function basis?  For example, maybe we provide some direction about what attributes should be set.
```
        trace.get_current_span().set_attributes(
            {"synapse.file_handle_id": file_result["fileHandleId"]}
        )
```

Reply:
I added more about this in https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1007/commits/0c91c295de5071ef5a01f3ef78264caefb293f29